src/components/AntdPopconfirm/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from "react";
+import styled from "@emotion/styled";
+import Checkbox from "@leafygreen-ui/checkbox";
+import { Popconfirm as AntPopconfirm } from "antd";
+import { size } from "constants/tokens";
+
+interface Props extends React.ComponentProps<typeof AntPopconfirm> {
+  children: React.ReactNode;
+}
+
+export const Popconfirm: React.VFC<Props> = ({ children, ...props }) => (
```

Comment:
I would be a little nervous to make all of those changes here. I think this could be handled by the existing ticket [EVG-18525](https://jira.mongodb.org/browse/EVG-18525).